tensorflow/python/keras/layers/embeddings_test.py

```diff
@@ -92,5 +92,23 @@ def test_eager_gpu_cpu(self):
     opt.apply_gradients(zip(gs, l.weights))
     self.assertAllEqual(len(gs), 1)
 
+  @tf_test_util.run_in_graph_and_eager_modes(use_gpu=False)
+  def test_embedding_invalid(self):
+    # len(input_length) should be equal to len(input_shape) - 1
+    with self.assertRaises(ValueError):
```

Comment:
Please check that the expected error message in being raised (otherwise the test could pass due to an unrelated exception).